WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -823,28 +838,35 @@ private extension DashboardViewController {
     }
 
     func observeBottomJetpackBenefitsBannerVisibilityUpdates() {
-        Publishers.CombineLatest(ServiceLocator.stores.site, $dashboardUI.eraseToAnyPublisher())
-            .sink { [weak self] site, dashboardUI in
-                guard let self = self else { return }
+        Publishers.CombineLatest3(ServiceLocator.stores.site,
+                                  $dashboardUI.eraseToAnyPublisher(),
+                                  viewModel.$showOnboarding)
```

Comment:
❓ similar to above, I don't think the bottom banner is affected from my understanding.